templates/core/index.html

```diff
@@ -1143,64 +447,272 @@ <h3 class='p-heading--4'><a href='engage/ubuntu-core-and-kura' onclick="dataLaye
       <!-- rtp section left end -->
     </div>
     <div class="col-4 p-divider__block">
       <!-- rtp section center start -->
-      <div class='p-heading-icon--muted'>
+      <div class='p-heading-icon--small'>
         <div class='p-heading-icon__header'>
           {{ image (
           url="https://assets.ubuntu.com/v1/6e184942-Webinar.svg",
           alt="",
           width="88",
           height="72",
           hi_def=True,
-          loading="lazy"
+          loading="lazy",
+          attrs={"class": "p-heading-icon__img"}
           ) | safe
           }}
-          <h4 class='p-heading-icon__title'>Webinar</h4>
+          <p class='p-heading-icon__title'>Webinar</p>
         </div>
       </div>
-      <h3 class='p-heading--4'><a href="https://www.brighttalk.com/webcast/6793/373030/migrating-to-ubuntu-core" onclick="dataLayer.push({'event' : 'GAEvent' , 'eventCategory' : 'product page' , 'eventAction' : 'core - center' , 'eventLabel' : 'Webinar - Migrating to Ubuntu Core' , 'eventValue' : '1' });">Migrating to Ubuntu Core</a></h3>
+      <h3 class='p-heading--4'><a href="https://www.brighttalk.com/webcast/6793/535454" onclick="dataLayer.push({'event' : 'GAEvent' , 'eventCategory' : 'product page' , 'eventAction' : 'core - center' , 'eventLabel' : 'Webinar - Yocto or Ubuntu Core?' , 'eventValue' : '1' });">Yocto or Ubuntu Core?&nbsp;&rsaquo;</a></h3>
       <!-- rtp section center end -->
     </div>
   </div>
 </section>
 
+<section class='p-strip'>
+  <div class='row u-vertically-center'>
+    <div class='col-8'>
+      <h2>Tamper-resistant and hardened</h2>
+      <p class='p-heading--4 u-no-margin--bottom'>Every aspect of the system is checked and verified.</p>
+    </div>
+  </div>
+  <div class="row u-vertically-center">
+    <div class="col-6">
+      <p>You need to know your software is pristine; not just for installation, but for the lifetime of the device.</p>
+      <p>Immutable packages and persistent digital signatures mean Ubuntu Core can verify any software component at any time, to guard against corruption and attack.</p>
+      <p>
+        <a href="/core/features/full-disk-encryption">Read an independent security audit of Ubuntu Core&nbsp;&rsaquo;</a>
+      </p>
+    </div>
+    <div class="col-6 u-hide--medium u-hide--small u-align--center u-vertically-center">
+      {{
+        image(
+        url="https://assets.ubuntu.com/v1/ad33af0c-Shield.svg",
+        alt="",
+        height="200",
+        width="200",
+        hi_def=True,
+        loading="lazy"
+        ) | safe
+      }}
+    </div>
+  </div>
+</section>
+  
+
+<section class='p-strip--image p-banner-core is-deep' id="built-for-business">
+  <div class='row u-vertically-center'>
```

Comment:
There's a lot of instances of single quotes used for attribute values - we should stick with double quotes for consistency and code cleanliness.

```suggestion
  <div class="row u-vertically-center">
```